### ISSUE TITLE
Error when installing Kubernetes clusters

### ISSUE THREAD

**Issue by shawtao** [weaveworks/wks-quickstart-firekube#98]
`• Using git branch: master
• Using git remote: origin

• Found jk 0.3.0
• Found footloose 0.6.3
• Found ignite 0.7.1
• Found wksctl 0.10.2
• Creating footloose manifest
• Creating SSH key
• Creating virtual machines
INFO[0000] Docker Image: weaveworks/ignite-centos:firekube-pre3 present locally 
INFO[0000] Creating machine: firekube-node0 ...         
INFO[0000] Machine firekube-node0 is already created... 
INFO[0000] Creating machine: firekube-node1 ...         
INFO[0000] Machine firekube-node1 is already created... 
• Creating Cluster API manifests
• Updating container images and git parameters
• Pushing initial cluster configuration
Everything up-to-date
• Installing Kubernetes cluster
FATA[2021-05-23T14:03:39+08:00] Error parsing manifest: failed to parse cluster manifest: for scheme unrecognized API group: cluster.k8s.io. Cause: UnknownGroup. gvk: cluster.k8s.io/v1alpha1, Kind=Cluster. error: no kind "Cluster" is registered for version "cluster.k8s.io/v1alpha1" in scheme "pkg/runtime/scheme.go:100" `

**Comment by mlejva** [comment]
Hey @shawtao, did you manage to solve this issue? I'm trying to run Firekube but running into the same problem

```
Error parsing manifest: failed to parse cluster manifest: for scheme unrecognized API group: cluster.k8s.io. Cause: UnknownGroup. gvk: cluster.k8s.io/v1alpha1, Ki
nd=Cluster. error: no kind "Cluster" is registered for version "cluster.k8s.io/v1alpha1" in scheme "pkg/runtime/scheme.go:100" 
```

Can we reopen this?

**Comment by mkell43** [comment]
@mlejva I ran into this issue as well and after some poking around, I believe I figured it out.

1. Edit `cluster.yaml` and replace `apiVersion: cluster.k8s.io/v1alpha1` with `apiVersion: cluster.x-k8s.io/v1alpha3`.
2. Edit `setup.js` and replace `apiVersion: 'cluster.k8s.io/v1alpha1',` with `apiVersion: 'cluster.x-k8s.io/v1alpha3'`.
3. Git add, commit, and push up those changes to your forked version of this repository.
4. Re-run `./setup.sh` or `wksctl apply` (which is the command that this failure was occurring on).
